src/frontend/js/types/commonDataProps.ts

```diff
@@ -23,7 +23,6 @@ enum FEATURES {
 
 export interface RichieContext {
   authentication: AuthenticationBackend;
-  csrftoken: string;
```

Comment:
Do not forgot to remove this property from the context factory.

https://github.com/openfun/richie/blob/a55cc68d79acbf75a97af1c6f020bcc107d62151/src/frontend/js/utils/test/factories/richie.ts#L155